target_chains/ethereum/examples/coin_flip/README.md

```diff
@@ -16,8 +16,13 @@ These commands will build dependencies for the typescript project.
 Next, choose a network to run the example on.
 The example has been deployed on the following networks:
 
-| Chain Name | Address | RPC |
-| optimism-goerli | 0x075A5160FF6462924B4124595F6f987187496476 | https://goerli.optimism.io |
+```
```

Comment:
tables don't format properly in github :(

Reply:
You can actually do this with html and there’s some examples out there or GitHub flavoured markdown mixed with HTML if you really want it these.